service-front/module/Application/src/View/Twig/AppFiltersExtension.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\View\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+class AppFiltersExtension extends AbstractExtension
+{
+    public function getFilters(): array
+    {
+        return [
+            new TwigFilter('ordinal_suffix', [$this, 'ordinalSuffix']),
+        ];
+    }
+
+    public function ordinalSuffix(int $number): string
```

Comment:
You might be able to just use Twig's number formatter for this. https://twig.symfony.com/doc/3.x/filters/format_number.html